server/routes/activities/record-attendance/handlers/activities.test.ts

```diff
@@ -373,7 +373,8 @@ describe('Route Handlers - Activities', () => {
           date: dateString,
           sessionFilters: 'AM,PM',
           categoryFilters: 'SAA_EDUCATION,SAA_INDUSTRIES',
-          locationFilters: 'IN_CELL,OUT_OF_CELL',
+          LocationId: '',
```

Comment:
lower "l"

Reply:
will put a note to fix in the next PR, it's not used in the test so will just remove it.